data/tddconfig-sim-ul.json

```diff
@@ -1,9 +1,21 @@
 {
- "symbol_size" : 1,
+ "antenna_num": 8,
+ "ue_num" : 8,
+ "core_offset": 1,
+ "worker_thread_num" : 22,
+ "socket_thread_num": 4,
+ "symbol_num_perframe": 14,
+ "frames" : [
+     "PUUUUUUUUUUUUU"
+ ],
+ "modulation": "64QAM",
+ "Zc":104,
+ "bs_server_addr": "192.168.21.181",
+ "bs_rru_addr": "192.168.21.182",
```

Comment:
Should we use the default addresses (127.0.0.1) here instead?

Reply:
Thanks for catching this. Fixed now.